src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,12 +1,21 @@
 <script setup>
 import {computed, ref} from 'vue'
 import {cleanDescription} from "@/utils/CleanDescription.js";
+import {filterArr} from "@/cleanupResources.js";
+import {MaBadge, MaInput} from "@mobileaction/action-kit";
+import "@mobileaction/action-kit/dist/style.css"
+
+const MAX_VALUE_OF_NGRAM_SIZE = 10;
+
+const filterSet = new Set(filterArr);
 
 const keywordText = ref('')
+const selectedNgramSizes = ref(new Array(MAX_VALUE_OF_NGRAM_SIZE).fill(false));
 
-const cleanKeywords = computed(() => {
-  return cleanDescription(keywordText.value).split(' ').filter(words => words.length > 0)
-})
+const toggleNgramCheckbox = function (ngramSize) {
```

Comment:
Please also convert functions to arrow functions
```suggestion
const toggleNgramCheckbox =  (ngramSize) => {
```